src/Clustering.jl

```diff
@@ -5,8 +5,6 @@
 
 module Clustering_
 
-export KMeans, KMedoids
```

Comment:
FYI these are not needed anymore, the `@mlj_model` macro does the export.

Reply:
Ok.